samples/webview-samples/multi-tab-browser/README.md

```diff
@@ -0,0 +1,40 @@
+<a target="_blank" href="https://chrome.google.com/webstore/detail/jhgiafbilglmngpgkhcmnicfkpfihggo">![Try it now in CWS](https://raw.github.com/GoogleChrome/chrome-app-samples/master/tryitnowbutton.png "Click here to install this sample from the Chrome Web Store")</a>
+
+
+# New Window API
+
+Sample that shows one way to combine the [New Window
+API](https://developer.chrome.com/apps/tags/webview#event-newwindow) with the
+[User Agent
+API](https://developer.chrome.com/apps/tags/webview#method-setUserAgentOverride)
+for
+[webviews](http://developer.chrome.com/apps/app_external.html#webview). The
+app combines the [New Window
+Sample](https://github.com/GoogleChrome/chrome-app-samples/tree/master/webview-samples/new-window)
+and the [User Agent
+Sample](https://github.com/GoogleChrome/chrome-app-samples/tree/master/webview-samples/user-agent)
+by supporting `Open link in new tab/window as...` via the [Context Menu
+API](https://developer.chrome.com/extensions/contextMenus) (see screenshot).
+
+## Features
+
+* All features from the [New Window
+Sample](https://github.com/GoogleChrome/chrome-app-samples/tree/master/webview-samples/new-window)
+* Context menu entries for `Open link in new tab/window
+  as Android/iPhone/Nokia Mobile/BlackBerry Playbook`
+
+## Limitations
+
+* See [New Window
+Sample](https://github.com/GoogleChrome/chrome-app-samples/tree/master/webview-samples/new-window)
```

Comment:
Done. I also propagated these changes to other folders under "/webview-samples/"